### ISSUE TITLE
A11y_AzureDataStudio_NewQuery_ScreenReader: Expand/Collapse state is not announced when we are expanding the "Select database" combo box using ctrl + downward arrow key

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#22671]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.3.1; #Linux; #ScreenReader; #ORCA; Element:Combobox;

### Actual Result:​
Expand/Collapse state is not announced when we are expanding the "Select database" combo box using ctrl + downward arrow key.

### Expected Result:​
Expand/Collapse state should be announced when we are expanding the "Select database" combo box using ctrl + downward arrow key.

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to "File" and hit ENTER key. TAB to "New Query" and hit ENTER key.
4. TAB to "Select database" and observe that Expand/Collapse state is not announced when we are expanding the "Select database" combo box using ctrl + downward arrow key.

### User Impact:
Screen reader users will not be able to know the state of combo box as it is expanded or collapsed.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachments:
[Uploading 228850831-8807262b-7170-4449-bbad-a9ba7f20c2c1.webm…]()

**Comment by PoojaNamde** [comment]
Primary bug: https://github.com/microsoft/azuredatastudio-mysql/issues/182

**Comment by PoojaNamde** [comment]
Lowering the severity of the bug to Sev3
